apis/io-engine/protobuf/v1/replica.proto

```diff
@@ -78,11 +79,23 @@ message UnshareReplicaRequest {
   string uuid = 1;  // uuid of the replica
 }
 
+// Resize(expand or shrink) a replica. Only valid for normal lvol
```

Comment:
nit: shrink might not be a use case here. So maybe remove it just to be clear with the expectations?

Reply:
Since this is the interface between control and data plane, we might as well leverage shrink for something internally.